via/static/scripts/video_player/components/VideoPlayerApp.tsx

```diff
@@ -209,21 +214,87 @@ export default function VideoPlayerApp({
             placeholder="Search..."
             value={filter}
           />
-          <div className="flex flex-row">
+        </div>
+      </div>
+
+      <main
+        data-testid="app-layout"
+        className={classnames('w-full flex min-h-0', {
+          // Stack video over transcript.
+          'flex-col': !multicolumn,
+          // Video and transcript side-by-side, using up remaining vertical
+          // space in app-container (i.e. 100vh minus top-bar)
+          'flex-row grow h-full': multicolumn,
+        })}
+        ref={appContainerRef}
+      >
+        <div
+          data-testid="embedded-video-container"
+          className={classnames(
+            'flex flex-col',
+            {
+              // Allow video to grow with available horizontal space in
+              // multicolumn layouts (NB: It will take up full width in
+              // single-column)
+              grow: multicolumn,
+            },
+            {
+              // Adapt spacing around video for different app sizes
+              'p-0': appSize === 'sm',
+              'p-1': appSize === 'md',
+              'p-2': appSize === 'lg',
+              'py-2 px-3': appSize === 'xl',
+              'py-2 px-4': appSize === '2xl',
+            }
+          )}
+        >
+          <YouTubeVideoPlayer
+            videoId={videoId}
+            play={playing}
+            time={timestamp}
+            onPlayingChanged={setPlaying}
+            onTimeChanged={setTimestamp}
+          />
+        </div>
+        <div
+          data-testid="transcript-container"
+          className={classnames(
+            'flex flex-col bg-grey-0 border-x',
+            {
+              // Make transcript fill available vertical space in single-column
+              // layouts
+              'min-h-0 grow': !multicolumn,
+              // TODO: This is a stopgap measure to prevent controls from being
+              // interfered with (overlaid) by sidebar controls and toolbar
+              'mr-[30px]': multicolumn,
+            },
+            {
+              // Adapt transcript width for different app sizes
+              'w-[480px]': appSize === '2xl',
+              'w-[450px]': appSize === 'xl',
+              'w-[410px]': appSize === 'lg',
+              'w-[380px]': appSize === 'md',
+            }
+          )}
+        >
+          <div
+            data-testid="transcript-controls"
+            className={classnames(
+              // Echo height of top-bar
```

Comment:
Can you expand this to clarify what "top-bar" is referring to here.

Reply:
It should just be worded "Same height as top-bar". I'm updating the comment.